lex_test.go

```diff
@@ -315,6 +315,73 @@ var lexFixtures = []lexFixture{
 		{"}", 39},
 		{"}", 40},
 	}},
+	{"lua-block-cert-slim", []tokenLine{
+		{"http", 1},
+		{"{", 1},
+		{"server", 2},
+		{"{", 2},
+		{"ssl_certificate_by_lua_block", 3},
+		{"{", 3},
+		{"print(\"Test lua ssl certificate!\")", 4},
+		{"}", 5},
```

Comment:
```
{"\n            print(\"Test lua ssl certificate!\")\n        ",5},
		{";", 5},
```
We grab everything in the Lua block as a single token and drop the {}. Example:
`content_by_lua_block {some code} `will be `content_by_lua_block  "some code";`  "some code" is an argument of directive.